### PR TITLE
Propose blog to introduce csi-cloning in 1.15

### PR DESCRIPTION
**Pull request by j-griffith**
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 

Moving this from dev-1.15 and resubmitting cherry pick of it onto master as per review requests.

### CODE REVIEW

File: content/en/blog/_posts/2019-06-19-volume-cloning-alpha.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+## What are the limitations of alpha?
+
+The alpha implementation of cloning for Kubernetes has the following limitations:
+
+* Does not support cloning volumes across different namespaces
+* Does not support cloning volumes across different storage classes (backends)
+
+## Future
+Depending on feedback and adoption, the Kubernetes team plans to push the CSI cloning implementation to beta in either 1.16.
```
Comment (sftim) [inline content/en/blog/_posts/2019-06-19-volume-cloning-alpha.md]: > in either 1.16.

Should there be some more text here?

Comment (j-griffith) [reply content/en/blog/_posts/2019-06-19-volume-cloning-alpha.md]: That's the second time I've done that with this PR, thanks @sftim, fixed.

File: content/en/blog/_posts/2019-06-19-volume-cloning-alpha.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+---
+layout: blog
+title: 'Introducing Volume Cloning Alpha for Kubernetes'
+date: 2019-06-19
```
Comment (msau42) [inline content/en/blog/_posts/2019-06-19-volume-cloning-alpha.md]: I think the date needed to be changed to 20?

**Comment by j-griffith** [comment]
@msau42 I've pushed the date out from the 19'th to the 20'th

**Comment by j-griffith** [comment]
@castrojo Update for Friday, fingers crossed the stars align :)

**Comment by castrojo** [comment]
/lgtm
/hold

Awesome thanks! We'll hold and merge on Friday, thanks!

**Pull request merged**